src/Messages/Attachments/TSAttachmentStream.m

```diff
@@ -34,10 +39,27 @@ - (instancetype)initWithPointer:(TSAttachmentPointer *)pointer
 
     _contentType = pointer.contentType;
     _isDownloaded = YES;
+    // TSAttachmentStream doesn't have any "incoming vs. outgoing"
+    // state, but this constructor is used only for new incoming
+    // attachments which don't need to be uploaded.
+    _isUploaded = YES;
 
     return self;
 }
 
+- (void)upgradeFromAttachmentSchemaVersion:(NSUInteger)attachmentSchemaVersion
```

Comment:
Yeah - it never occurred to me to do this, but I think I like it.

Now I'm wondering if we should just have a unified schema version across all our TSYapDatabaseObjects instead of tracking per model 👀 ...